disable-comments.php

```diff
@@ -235,7 +235,11 @@ public function filter_wp_headers( $headers ) {
 	 */
 	public function filter_query() {
 		if( is_comment_feed() ) {
-			wp_die( __( 'Comments are closed.' ), '', array( 'response' => 403 ) );
+			$refere = isset($_SERVER['HTTP_REFERER'])? $_SERVER['HTTP_REFERER']: '';
+			$location = !empty($refere)? htmlspecialchars($refere): home_url('/');
```

Comment:
This function actually used to redirect before. It was changed here: https://github.com/solarissmoke/disable-comments/commit/447be0a3fa883650c29140d99dd531ba66f783eb because of a security issue that was introduced by the redirect logic.

I agree that the wp_die message is pretty ugly. However this is a feed URL which real users are unlikely to see, so I don't think it is a big problem?